pages/[chain]/collection/[contract].tsx

```diff
@@ -155,6 +163,94 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
     fallbackData: initialTokenFallbackData ? [ssr.tokens] : undefined,
   })
 
+  useTokenUpdateStream(id as string, collectionChain.id, {
+    onClose: () => setSocketState(0),
+    onOpen: () => setSocketState(1),
+    onMessage: ({
+      data: reservoirEvent,
+    }: MessageEvent<ReservoirWebsocketIncomingEvent>) => {
+      let hasChange = false
+
+      const newTokens = [...tokens]
```

Comment:
check if filtered by an attribute, if so we just need to not make any changes, since we don't know the attributes. We should also make sure to hide the connected indicator since we're effectively ignoring updates.